synapse/storage/data_stores/main/schema/full_schemas/README.md

```diff
@@ -1,13 +1,20 @@
-# Building full schema dumps
+# Synapse Database Schemas
 
-These schemas need to be made from a database that has had all background updates run.
+These schemas are used to set up a brand new Synapse DB on both SQLite3 and
+Postgres
 
-To do so, use `scripts-dev/make_full_schema.sh`. This will produce
-`full.sql.postgres ` and `full.sql.sqlite` files.
+## Building full schema dumps
+
+If you want to recreate these schemas, they need to be made from a database that
+has had all background updates run.
+
+To do so, use `scripts-dev/make_full_schema.sh`. This will produce new
+`full.sql.postgres ` and `full.sql.sqlite` files. 
 
 Ensure postgres is installed and your user has the ability to run bash commands
-such as `createdb`.
+such as `createdb`, then call
+
+    ./scripts-dev/make_full_schema.sh -p postgres_username -o output_dir/
 
-```
-./scripts-dev/make_full_schema.sh -p postgres_username -o output_dir/
-```
+There are two folders with full-schema snapshots `16` is a snapshot from 2015,
+for historical reference, and one folder with most recent database-schema.
```

Comment:
```suggestion
for historical reference. `54` is currently the most recent full schema snapshot.
```

Reply:
I'll change it to "The other contains the most recent full
schema snapshot." so we wouldn't have to change the Readme every time, we update the schema

Reply:
but this will be wrong next time we create a full schema... :man_shrugging:

Reply:
Why is this wron the next time? "the other" is always the latest, isn't it?

Reply:
yes, but then there will be three folders.

Reply:
the point is: we're going to have to update this paragraph next time we create a full schema anyway, so we may as well be clear which folders we are talking about.